cypress/integration/twilio-video.spec.js

```diff
@@ -82,6 +82,38 @@ context('A video app user', () => {
       cy.getParticipant('test1').should('not.exist');
       cy.get('[data-cy-main-participant]').should('contain', 'testuser');
     });
+
+    describe('the recording start/stop feature', () => {
+      before(() => {
+        cy.get('[data-cy-more-button]')
+          .last()
```

Comment:
Oh I see. `.last()` is needed because there's a "More" button in the top menu bar (on mobile), and the bottom menu bar (on desktop). 

An alternative way to select the right "More" button is to use the ["descendant combinator"](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) to select the button that exists in the `footer` element:

`cy.get('footer [data-cy-more-button]').click()`.

Not sure which is better here. I think `footer [data-cy-more-button]` is a little more specific, and would be better if there was a chance that the two buttons would change order, but there isn't a chance of that happening in this app. I'll let you decide. 🙂

Reply:
cool, thanks! i didn't know that you could do that in cypress. i went with your suggestion because i agree that it is more specific, but it will also help me remember that i could use this syntax in the future 👍